src/include/catalog/schema.h

```diff
@@ -134,15 +158,19 @@ class Schema {
     uint16_t max_varlen_size_;
     bool nullable_;
     col_oid_t oid_;
-    // TODO(Matt): default value would go here
-    // Value default_;
+    // TODO(Sai): Consider having a DefaultValueObject containing isNull, 16-byte variable and attribute size
+    // This avoids handling memory explicitly for default values
+    byte default_[16];
```

Comment:
Why is 16 chosen? Do you plan to change it to byte* in the future?

Reply:
Maybe 16 is chosen for a reason, but you should not simply use it as a magic number.